templates/base.html

```diff
@@ -263,7 +263,7 @@ <h4 class="modal-title"></h4>
                         <li class="not-found"><a>{% trans "No matches" %}</a></li>
                     </ul>
                 </span>
-                <input type="text" class="form-control" placeholder="{% trans 'Search for...' %}" />
+                <input type="text" aria-label="search" class="form-control" placeholder="{% trans 'Search for...' %}" />
```

Comment:
The `aria-label` value should be translated. You can use the same value as the `placeholder` attribute so that there is no new string in the po file.